cfme/tests/configure/test_default_filters.py

```diff
@@ -13,70 +11,63 @@
     host_collecton = appliance.collections.hosts
-    navigate_to(host_collecton, 'All')
-    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+    view = navigate_to(host_collecton, 'All')
+    assert view.filters.navigation.has_item('Platform / HyperV'), 'Default Filter settings Failed!'
 
 
-def test_infrastructurevms_defaultfilters():
+def test_infrastructurevms_defaultfilters(appliance):
     filters = [['Infrastructure', 'Virtual Machines', 'VMs', 'Platform / VMware']]
-    df = st.DefaultFilter(name='Platform / VMware')
-    df.update({'filters': [(k, True) for k in filters]})
-    navigate_to(vms.Vm, 'VMsOnly')
-    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+    tree_path = ['All VMs', 'Global Filters', 'Platform / VMware']
+    appliance.user.my_settings.default_filters.update({'filters': [(k, True) for k in filters]})
+    view = navigate_to(vms.Vm, 'VMsOnly')
+    assert view.sidebar.vms.tree.has_path(*tree_path), 'Default Filter settings Failed!'
 
 
-def test_infrastructuretemplates_defaultfilters():
+def test_infrastructuretemplates_defaultfilters(appliance):
     filters = [['Infrastructure', 'Virtual Machines', 'Templates', 'Platform / Redhat']]
-    df = st.DefaultFilter(name='Platform / Redhat')
-    df.update({'filters': [(k, True) for k in filters]})
-    navigate_to(vms.Template, 'TemplatesOnly')
-    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+    tree_path = ['All Templates', 'Global Filters', 'Platform / Redhat']
+    appliance.user.my_settings.default_filters.update({'filters': [(k, True) for k in filters]})
+    view = navigate_to(vms.Template, 'TemplatesOnly')
+    assert view.sidebar.templates.tree.has_path(*tree_path), 'Default Filter settings Failed!'
 
 
-@pytest.mark.meta(blockers=[BZ(1290300, forced_streams=["upstream"])])
-def test_servicetemplateandimages_defaultfilters():
+@pytest.mark.meta(blockers=[BZ(1290300, forced_streams=['upstream'])])
+def test_servicetemplateandimages_defaultfilters(appliance):
     filters = [['Services', 'Workloads', 'Templates & Images', 'Platform / Microsoft']]
-    df = st.DefaultFilter(name='Platform / Microsoft')
-    df.update({'filters': [(k, True) for k in filters]})
-    try:
-        view = navigate_to(workloads.TemplatesImages, 'All')
-        view.templates.select_global_filter(df.name)
-        assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
-    finally:
-        df.update({'filters': [(k, False) for k in filters]})
+    tree_path = ['All Templates & Images', 'Global Filters', 'Platform / Microsoft']
+    appliance.user.my_settings.default_filters.update({'filters': [(k, True) for k in filters]})
+    view = navigate_to(workloads.TemplatesImages, 'All')
+    assert view.templates.tree.has_path(*tree_path), 'Default Filter settings Failed!'
+    appliance.user.my_settings.default_filters.update({'filters': [(k, False) for k in filters]})
```

Comment:
**Required**: Use request.addfinalizer here as this will not run if the assert fails. Please update other places where needed